modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html

```diff
@@ -292,7 +292,7 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 [#if mdf_claim_id?has_content]
 	[#assign activities = restClient.get("/c/mdfclaims/${mdf_claim_id}/mdfClmToMDFClmActs?nestedFields=mdfClmActToMDFClmBgts,mdfClmActToMDFActDocs").items /]
 
-	[#list activities?filter(activity -> activity.selected == true) as activity]
+	[#list (activities?filter(activity -> activity.selected)! == true) as activity]
```

Comment:
```suggestion
	[#list (activities?filter(activity -> activity.selected))! == true) as activity]
```

Reply:
I tried this way:

[#list ((activities?filter(activity -> activity.selected))! == true) as activity]